dev/io.openliberty.checkpoint_fat_microprofile/.project

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>io.openliberty.checkpoint_fat_microprofile</name>
```

Comment:
The folder/project name should be `io.openliberty.checkpoint_fat_mpconfig`.  I don't think we will add other microprofile tests to this bucket.